misc/scripts/install-local.sh

```diff
@@ -225,8 +225,11 @@ fi
 if [[ -n $build_depends ]]; then
     echo "build_dependencies=\"$build_depends"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
 fi
-if [[ -n $pacdepslist ]]; then
-    echo "pacdeps=\"${pacdepslist[*]}"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
+if [[ -n $pacdeps ]]; then
+    echo "pacdeps=\"$pacdeps"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
+fi
```

Comment:
```suggestion
if [[ -n $pacdeps ]]; then
    echo "_pacdeps=\"$pacdeps"\" | sudo tee -a /var/log/pacstall_installed/"$PACKAGE" >/dev/null
fi
```
If I understand correctly this portion's `pacdeps` is the pacscript pacdeps, which is fine, What I am asking you to do is write it to the file using the `_` prefix.

Reply:
oh